pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -515,7 +457,7 @@ func (r *ReconcileChaosEngine) forceRemoveAllChaosPods(engine *chaosTypes.Engine
 	}
 	if err != nil {
 		r.recorder.Eventf(engine.Instance, corev1.EventTypeWarning, "ChaosResourcesOperationFailed", "(chaos stop) Unable to delete chaos resources: %v allocated to chaosengine", strings.Join(deleteEvent, ""))
-		return fmt.Errorf("Unable to delete ChaosResources due to %v", err)
+		return fmt.Errorf("unable to delete ChaosResources due to %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("unable to delete ChaosResources due to %w", err)
```

Reply:
golint is giving a warning after replacing %v by %w for the cases where error args is a list
`Errorf format %w has arg err of wrong type []error`

Reply:
Seems like this is a slice, please collect all the errors and append them into one `error`, and then try to use `%w` directive.